frontend/pages/login.tsx

```diff
@@ -0,0 +1 @@
+export { default } from 'app/bookshelf-login'
```

Comment:
Do you still need the `pages` directory? Iirc Next.js 13 is starting to do away this folder and put everything in the `app` folder